goon/code/datums/browserOutput.dm

```diff
@@ -175,7 +175,7 @@ For the main html chat area
 			//Uh oh this fucker has a history of playing on a banned account!!
 			if (found.len > 0)
 				//TODO: add a new evasion ban for the CURRENT client details, using the matched row details
-				message_admins("[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])")
+				message_admins("<FONT SIZE=+2 COLOR=red>[key_name(src.owner)] has a cookie from a banned account! (Matched: [found["ckey"]], [found["ip"]], [found["compid"]])</font>")
```

Comment:
Might as well make it send this alert to discord. Also please avoid using COLOR=red, there's already the danger class for red+bold.

Reply:
I'd suggest not sending it to Discord until its tendency to report that a person is themselves is fixed

Reply:
We get a lot of false positives so I'm not sure a discord send would be good at this point but should that ever be fixed I could definitely work on getting it to send to discord.
As for the color, what do you suggest I change? Just use the hex code for red or some other color?

Reply:
`<span class='danger'>`